app/service/update-appeal-service.ts

```diff
@@ -92,6 +115,19 @@ export default class UpdateAppealService {
     await this.ccdService.updateCase(currentUserId, updatedCcdCase, securityHeaders);
   }
 
+  async submitAppeal(req: Request) {
+    const securityHeaders: SecurityHeaders = await this.getSecurityHeaders(req);
+
+    const currentUserId = req.idam.userDetails.id;
+    const caseData = this.convertToCcdCaseData(req.session.appeal.application);
+    const updatedCcdCase = {
+      id: req.session.ccdCaseId,
+      case_data: caseData
+    };
+
+    await this.ccdService.submitAppeal(currentUserId, updatedCcdCase, securityHeaders);
+  }
+
```

Comment:
I guess we dont need this method anymore as we are merging ccdService submit and update into just one method?